src/options-support.js

```diff
@@ -20,4 +22,18 @@ function getOptions() {
   }
 }
 
-module.exports = { parseTransformArgs, getOptions };
+function extractJSCodeShiftOptions(options, codeShiftOptions = jsCodeShiftOptions) {
+  let cliOptions = Object.assign({}, options);
+  let jsCodeShiftOptions = [];
+  codeShiftOptions.forEach(option => {
+    if (Object.prototype.hasOwnProperty.call(cliOptions, option)) {
+      let camelCaseOption = option.replace(/-([a-z])/g, (_, up) => up.toUpperCase());
+      jsCodeShiftOptions.push(`--${option}`, options[option]);
+      delete cliOptions[option];
+      delete cliOptions[camelCaseOption];
+    }
+  });
+  return { cliOptions, jsCodeShiftOptions };
+}
```

Comment:
I think I'd prefer to do this all in `parseTransformArgs`, and ahve that essentially `return { paths, options, transformerOptions }` or something like that.

What do you think?

Reply:
Hi, this is my first open-source contribution that's why I was little hesitant towards altering the existing function and breaking the existing flow. Anyways, I have incorporated the changes in the parseTransformArgs itself.
Kindly have a look and let me know if this is okay.